core-services/egov-enc-service/src/main/resources/db/migration/main/V20230930120401__eg_enc.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS  eg_enc_mdms_config;
+
+CREATE TABLE public."eg_enc_mdms_config"
+(
+    id SERIAL,
+    tenant_id text NOT NULL,
+    PRIMARY KEY (id)
+);
+
```

Comment:
How will the user add tenants when they select DB implementation?

Reply:
They will have to add them manually, similar to how we add in MDMS JSON.